tests/tests_aqua.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import requirements_parser
+import os
+
+requirements_parser = requirements_parser.RequirementsParser()
+
+
+class MyTestCase(unittest.TestCase):
+
+    def test_requirements_num(self):
+        """check result if double item and the functions (with numbers) and check when requirements after test ('test_requirements.txt')"""
+        dict_result = {"1", "2", "3", "4", "5", "6", "7"}
+
+        with open('test_requirements.txt', 'w') as file:
+            file.write("1\n 2\n 3\n -r test2_requirements.txt\n 4\n")
+
+        with open('test2_requirements.txt', 'w') as file2:
+            file2.write("1\n 5\n 6\n 7\n")
+
+        output_dict = requirements_parser.open_requirements('test_requirements.txt')
+
+        os.remove('test_requirements.txt')
+        os.remove('test2_requirements.txt')
+
+        self.assertEqual(output_dict, dict_result)
+
+    def test_requirements_letters(self):
```

Comment:
you do not need this test. is the same test as before just with letters. you need to check different use cases and not types.